packages/calypso-e2e/src/secrets/secrets-manager.ts

```diff
@@ -35,6 +35,10 @@ export interface Secrets {
 	storeSandboxCookieValue: string;
 	testCouponCode: string;
 	wpccAuthPath: string;
+	wpApiSecrets: {
```

Comment:
Tiny nit, but would you be okay if we renamed this property to be a bit more specific to what it is? This is really the Calypso Oauth Application. Maybe we could call it `calypsoOauthApplication`?

`wpApiSecrets` could be interpreted as some kind of admin API key which might confuse folks!

Reply:
Yeah, that sounds good to me. I was at a bit of a loss as to what I should call it so the `wpApiSecrets` was more of a placeholder.

Reply:
Fixed in c572c75b88c3213cf885b21230bdf1d732f0da23.